Cargo.toml

```diff
@@ -54,7 +55,7 @@ dxt = ["image/dxt", "embedded_images"]
 dds = ["image/dds", "embedded_images"]
 webp = ["image/webp", "embedded_images"]
 # enables svg
-svg = ["svg2pdf", "usvg", "pdf-writer"]
+svg = ["svg2pdf", "usvg", "pdf-writer", "fontdb"]
```

Comment:
I'd move this into a separate `svg_enable_fontdb` feature and do `if cfg!(feature = svg_enable_fontdb) {}` in the code later